examples/static_graph_optimizations/ptb/README.md

```diff
@@ -0,0 +1,9 @@
+# Recurrent Net Language Model
```

Comment:
Add `with static subgraph optimization` to make clear how this is different from the other one.